src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
 // write code here
+
+const items = document.querySelector('li');
+const tree = document.querySelector('.tree');
+
+for (const item of items) {
+  const span = document.querySelector('span');
```

Comment:
The `span` element is being selected inside the loop. This will select the first `span` element in the document for each `li`. Consider creating a new `span` element for each `li` instead.